warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_stale_vehicle_positions.sql

```diff
@@ -9,22 +9,27 @@
 ) }}
 
 {% if is_incremental() %}
-    {% set timestamps = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set timestamps = dbt_utils.get_column_values(table=this, column='date', order_by = 'date DESC', max_records = 1) %}
     {% set max_ts = timestamps[0] %}
 {% endif %}
 
 WITH
 
 feed_guideline_index AS (
     SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index_vp') }}
+    {% if is_incremental() %}
+    WHERE date >= EXTRACT(DATE FROM TIMESTAMP('{{ max_ts }}'))
+    {% else %}
+    WHERE date >= DATE_SUB(CURRENT_DATE(), INTERVAL {{ var('RT_LOOKBACK_DAYS') }} DAY)
+    {% endif %}
 ),
```

Comment:
let me know if I have this wrong - but my sense is we want to filter the index table the same way we filter the vp table, so that the join doesn't result in many null days

Reply:
Yes I think this is correct